File: solutions/ocaml/darts/1/darts.ml
let score (x: float) (y: float): int = let n : float = x ** 2.0 +. y ** 2.0 in
  if n > 100.0 then 0 else
    if n > 25.0 then 1 else
      if n > 1.0 then 5 else 10

File: solutions/kotlin/darts/1/src/main/kotlin/Darts.kt
object Darts {

    fun score(x: Float, y: Float /* choose proper types! */): Int {
        val r2 = x*x + y*y
        return when {
            r2 > 100 -> 0
            r2 > 25 -> 1
            r2 > 1 -> 5
            else -> 10
        }
    }
}
